### ISSUE TITLE
UI: A11y: Live stream timecode is not accessible as button

### ISSUE THREAD

**Issue by joeyparrish** [google/shaka-player#1861]
**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**
Yes

**What version of Shaka Player are you using?**
master

**Can you reproduce the issue with our latest release version?**
N/A - no UI in v2.4.x

**Can you reproduce the issue with the latest code from `master`?**
yes

**Are you using the demo app or your own custom app?**
demo

**What browser and OS are you using?**
ChromeOS

**What are the manifest and license server URIs?**
Any live stream

**What did you do?**
1. Load live stream
2. Try to tab to the timecode

**What did you expect to happen?**
Since this is clickable (seeks to live edge), it should be tab-seekable and accessible as well.

**What actually happened?**
The timecode can't be tab-navigated, and it has no role and no aria label.  It is a div with a click event.

This should be accessible, ideally as a styled button (inherently accessible) instead of a div.  The aria label `ARIA_LABEL_LIVE` (already exists, unused) should be applied if `player.isLive()` to alert users that it can be used to seek to the live edge.  The same label could be used as a tooltip for sighted users, again if `player.isLive()`.
